api/package.json

```diff
@@ -18,9 +18,8 @@
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "es-main": "^1.2.0",
-    "express": "^4.18.2",
-    "express-async-errors": "^3.1.1",
```

Comment:
Were we not using express-async-errors before?

Reply:
Yep!  As mentioned in the PR body, we're migrating to V5 to avoid dealing with that lib.  Express v5 can handle async middlewares natively.